app-web/gatsby-config.js

```diff
@@ -19,7 +19,7 @@ const eventbritePlugin = () =>
       }
     : undefined;
 
-const devopsCommonsMeetup = () =>
+/*const devopsCommonsMeetup = () =>
```

Comment:
let's document why this is being commented out. Maybe link it to that rocket chat message?